docs/api-docs/management-apis/contextual-filters.md

```diff
@@ -0,0 +1,310 @@
+
+```json
+{
+    "data": [
+        {
+            "id": "YnJhbmQ=",
+            "type": "brand",
+            "name": "Brand",
+            "product_count": 3
+        },
+        {
+            "id": "cmF0aW5n",
+            "type": "rating",
+            "name": "Rating",
+            "product_count": 1
+        },
+        {
+            "id": "cHJpY2U=",
+            "type": "price",
+            "name": "Price",
+            "price_range_min": 9.95,
+            "price_range_max": 225
+        },
+        ...
+    ],
+    "meta": {}
+}
+```
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Categories
+> To get a list of your store's categories, send a `GET` request to [`/v3/catalog/categories`](https://developer.bigcommerce.com/api-reference/store-management/catalog/category/getcategories).
+
+</div>
+</div>
+</div>
+
+## Configure contextual filters
+
+Let's see if there are any contextual overrides present on a given category. To do so, send a `GET` request to [`/v3/settings/search/filters/contexts`](insert-link-here) specifying the `category_id` query parameter.
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/createwidgettemplate#requestrunner) // replace link
+
+The response will contain existing contextual overrides. 
+
+```json
+{
+    "data": [
+        {
+            "context": {
+                "channel_id": 1,
+                "category_id": 21
+            },
+            "data": [
+                {
+                    "id": "U2l6ZQ==",
+                    "display_name": "Size",
+                    "type": "product",
+                    "display_product_count": false,
+                    "collapsed_by_default": false,
+                    "items_to_show": 5,
+                    "sort_by": "alpha",
+                    "is_enabled": false
+                },
+                {
+                    "id": "cHJpY2U=",
+                    "display_name": "Price",
+                    "type": "price",
+                    "collapsed_by_default": true,
+                    "is_enabled": true
+                },
+                ...
+                {
+                    "id": "Ym9vbA==",
+                    "display_name": "Other",
+                    "type": "other",
+                    "is_enabled": true,
+                    "display_product_count": true,
+                    "collapsed_by_default": true,
+                    "show_free_shipping_filter": true,
+                    "show_is_featured_filter": true,
+                    "show_in_stock_filter": true
+                }
+            ]
+        },
+    ],
+    "meta": {}
+}
+```
+
+The order and names of returned filters should match the filters listed under **Category Filters** in the control panel.
```

Comment:
I think somewhere we need to mention that you change the order of the filters on the live site by changing the order of the data in the array. Essentially, the "sort order" of filters on the storefront is determined _implicitly_ from the order of the data you send.